packages/frames.js/src/next/server.tsx

```diff
@@ -35,6 +38,11 @@ export * from "./types";
 import { ImageResponse } from "@vercel/og";
 import type { SatoriOptions } from "satori";
 
+// this is ugly hack to go around the issue https://github.com/vercel/next.js/pull/61721
+const NextResponse = (
+  "default" in NextResponseBase ? NextResponseBase.default : NextResponseBase
+) as typeof NextResponseBase;
+
```

Comment:
@davidfurlong  this is an ugly hack to get around the next.js bug. This should also work work next 14.1.1. Don't like this but this is the only way I got debugger to work properly in ESM module scope.

Reply:
@stephancill also for your information.